packages/website/docs/components/patterns/SelectionAssistant/InputSelectionAssistant/Basic/sample.html

```diff
@@ -0,0 +1,39 @@
+<!-- playground-fold -->
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<style>
+		#btn {
+			display: none;
+			position: absolute;
+		}
+	</style>
+	<title>Sample</title>
+</head>
+
+<body style="background-color: var(--sapBackgroundColor); height: 300px;">
+	<!-- playground-fold-end -->
```

Comment:
is this correct?

Reply:
Seems to be the same for all the other samples.